server.py

```diff
@@ -45,6 +49,11 @@
     await ws.prepare(request)
     # WebSocket is added to a list
     ws_list.append(ws)
+    if ws_list == ws_interfaces:
+        assigned_robots[ws] = available_robots[0]
+        available_robots.pop(0)
```

Comment:
`pop()` vrací to co ze seznamu oddělá, takže tyhle dva řádky jdou spojit do jednoho.